internal/proftest/proftest.go

```diff
@@ -92,14 +93,16 @@ func (ui *TestUI) Print(args ...interface{}) {
 // error messages are permitted when appropriate.
 func (ui *TestUI) PrintErr(args ...interface{}) {
 	if ui.IgnoreRx != "" {
-		if matched, err := regexp.MatchString(ui.IgnoreRx, fmt.Sprint(args)); matched || err != nil {
+		if matched, err := regexp.MatchString(ui.IgnoreRx, fmt.Sprint(args...)); matched || err != nil {
 			if err != nil {
 				ui.T.Errorf("failed to match against regex %q: %v", ui.IgnoreRx, err)
 			}
+			ui.IgnoredErrCount++
 			return
 		}
 	}
 	if ui.Ignore > 0 {
+		ui.IgnoredErrCount++
```

Comment:
I would only expect the counter to be increased when the regex is matched. I think there is a naming issue. Perhaps IgnoreRx should be AllowRx and IgnoredErrCount should be NumAllowRxMatches or something like that.